src/collectd-snmp.pod

```diff
@@ -373,6 +373,13 @@ How long to wait for a response. The C<Net-SNMP> library default is 1 second.
 The number of times that a query should be retried after the Timeout expires.
 The C<Net-SNMP> library default is 5.
 
+=item B<Bulk_Size> I<Integer>
+
+This number activate and configure snmp bulk transfers. For B<Table> data it allow
```

Comment:
typo: allows̲

Reply:
"This number"...

This is not number, but option. Option description should be rephrased.

Reply:
Well, this is an option and a number size (0 = off, 100 = size of bulk transfert), however not beeing a native english speaker i welcome any rephrasing suggestion

Reply:
> Configures the size of SNMP bulk transfers. The default is 0, which disables bulk transfers altogether.

Reply:
Ok, not usefull to mention SNMP v2 or the fact that it's limited to table data for now ?